pkg/auth/dependency/newinteraction/nodes/verify_identity.go

```diff
@@ -0,0 +1,151 @@
+package nodes
+
+import (
+	"errors"
+
+	"github.com/authgear/authgear-server/pkg/auth/dependency/identity"
+	"github.com/authgear/authgear-server/pkg/auth/dependency/newinteraction"
+	"github.com/authgear/authgear-server/pkg/auth/dependency/verification"
+	"github.com/authgear/authgear-server/pkg/core/uuid"
+	"github.com/authgear/authgear-server/pkg/otp"
+)
+
+func init() {
+	newinteraction.RegisterNode(&NodeVerifyIdentity{})
+}
+
+type EdgeVerifyIdentity struct {
+	Identity *identity.Info
+}
+
+func (e *EdgeVerifyIdentity) Instantiate(ctx *newinteraction.Context, graph *newinteraction.Graph, rawInput interface{}) (newinteraction.Node, error) {
+	node := &NodeVerifyIdentity{
+		Identity: e.Identity,
+		ID:       uuid.New(),
+	}
+	result, err := node.SendCode(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	node.Channel = result.Channel
+	node.CodeLength = result.CodeLength
+	node.SendCooldown = result.SendCooldown
+	return node, nil
+}
+
+type NodeVerifyIdentity struct {
+	Identity     *identity.Info `json:"identity"`
+	ID           string         `json:"id"`
```

Comment:
The name `ID` is too vague. I suggest following the convention of forgot password and naming this `CodeHash`.

Reply:
`CodeHash` is inaccurate, since it is not a hash. I've changed it to `CodeID` in a later commit.